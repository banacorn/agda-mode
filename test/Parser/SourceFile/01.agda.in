data ℕ : Set where
  Z : ℕ
  S : ℕ → ℕ

-- the following ? should be expanded in into {!   !}
-- the comment above should not be affected
a = ?

_+_ : ℕ → ℕ → ℕ
Z + y = {!   !}
S x + y = {!   !}