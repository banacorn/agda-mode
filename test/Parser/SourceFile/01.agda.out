Hole [0] (142, 143) => (142, 149) "{!   !}"
Hole [1] (175, 182) => (175, 182) "{!   !}"
Hole [2] (193, 200) => (193, 200) "{!   !}"